configuration/rules-dsl.md

```diff
@@ -204,32 +204,29 @@ You can find it from PaperUI or from Karaf remote console.
 For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
-> The parser will not recognize the whole text correctly without quotes.
 
 {: #channel-based-triggers}
 ### Channel-based Triggers
 
-You can listen for triggers from specific *trigger channels* that are provided by some addons.
-You can decide whether you want to catch only a specific trigger or any.
-Here is the syntax for all these cases (parts in square brackets are optional):
+Some add-ons provide trigger channels. 
+Your rules can take actions based upon trigger events generated by these trigger channels.
+You can decide whether you want to catch only a specific or any trigger the channel provides.
+Here is the syntax for these cases (parts in square brackets are optional):
 
 ```java
 Channel "<triggerChannel>" triggered [<triggerEvent>]
 ```
 
-`triggerChannel` is the identifier for this specific channel.
-For example, the [Amazon Dash Button Binding]({{base}}/addons/bindings/amazondashbutton/readme.html#channels) provides the "press" trigger channel.
-This channel will be triggered, when a press is detected for a dash button.
-For a deeper dive into trigger channels you can take a look at the [Astro Binding]({{base}}/addons/bindings/astro/readme.html#trigger-channels).
-You can find a big variety of possible channels and events.
```

Comment:
Consider deleting the sentence, "You can find a big variety...", or modify to, "oir a deeper dive into trigger channels, please take a look at the [Astro Binding]({{base}}/addons/bindings/astro/readme.html#trigger-channels) where you can see an example of a binding with a variety of channels and events."

Reply:
The sentence was removed completely already with the whole paragraph.